flang/tools/bbc/bbc.cpp

```diff
@@ -325,11 +326,13 @@ int main(int argc, char **argv) {
   ProgramName programPrefix;
   programPrefix = argv[0] + ": "s;
 
-  if (includeDirs.size() == 0)
+  if (includeDirs.size() == 0) {
```

Comment:
At a glance, this looks like the current directory is *not* always searched.  Is there a reason to not automatically look in any explicitly specified directories, plus the current directory, plus one or two or more "system" directories in some order?  That would make it harder to short circuit that search sequence, although that could be done with an  option to suppress all default search directories.

Reply:
Correct. If the user of `bbc` is specifying search directories, the assumption is they know what they are doing.